CopilotKit/package.json

```diff
@@ -2,7 +2,7 @@
   "private": false,
   "scripts": {
     "build": "turbo build",
-    "dev": "turbo dev",
+    "dev": "turbo dev --concurrency 11",
```

Comment:
note for future: dev build complains that more concurrency is required to run this